sickbeard/providers/torrentproject.py

```diff
@@ -101,6 +101,8 @@ def search(self, search_strings, age=0, ep_obj=None):  # pylint: disable=too-man
 
                     t_hash = torrents[i]["torrent_hash"]
                     torrent_size = torrents[i]["torrent_size"]
+                    if not all([t_hash, torrent_size]):
+                        continue
                     size = convert_size(torrent_size) or -1
```

Comment:
So are you discarding result because of that?
When no size it should set as -1
Maybe when no hash set as None